src/rt/aaA.d

```diff
@@ -485,6 +485,14 @@ pure nothrow @nogc unittest
 // API Implementation
 //------------------------------------------------------------------------------
 
+/// Allocate new AA implementation.
```

Comment:
Please add documentation like e.g. https://github.com/dlang/druntime/pull/3865 (cc @dkorpel)
In particular, mention when this function is called by  the compiler.

Reply:
Updated docs.